content/docs/learning-guide/tutorials/animation/data-driven-root-motion.md

```diff
@@ -0,0 +1,152 @@
+
+The following movie demonstrates a forward run cycle created for code driven root motion:
+
+{{< video src="/images/learning-guide/tutorials/animation/jack-run-no-root-motion.mp4" info="Example run cycle without root motion." autoplay="true" loop="true" poster="/images/learning-guide/tutorials/animation/jack-run-poster.png" >}}
+
+Notice that the actor runs in place. At runtime, when a forward input event is received, the script plays the preceding forward run cycle and moves the actor entity forward based on the result of some expression or function.
+
+### Data driven
+
+For data driven root motion, motion data is provided that moves the actor off the scene origin. The data is usually keyframe animation provided by an animator. The following movie demonstrates a forward run cycle created for data driven root motion:
+
+{{< video src="/images/learning-guide/tutorials/animation/jack-run-root-motion.mp4" info="Example run cycle with root motion." autoplay="true" loop="true" poster="/images/learning-guide/tutorials/animation/jack-run-poster.png" >}}
+
+Notice that the actor is moving forward as it runs. At runtime, when a forward input event is received, the forward run cycle plays. The animation data from the root of the actor's skeleton drives the forward movement of the actor entity.
+
+{{< note >}}
+The preceding example movies display the actor's bones as green lines. The green line that extends from the ground plane to the actor's pelvis is the root bone of the skeleton. In an ideal scenario, actor assets should have this root bone, and any animation that moves the actor off of the scene origin should be applied to this root bone.
+{{< /note >}}
+
+### Which should you use?
+
+Whether you use code driven or data driven root motion depends on your project's needs.
+
+Code driven root motion might be preferable in scenarios where precise actor movement for play mechanics is more critical than visual performance. Platform games and fighting games, for example, usually require accurate actor movement. Ensuring an actor moves a precise distance in response to an input might be far more important than ensuring actor's movement relative to animation is visually accurate.
+
+Data driven root motion can give animators more control over the actor's performance. Adventure games and first person games that strive for realism and immersion often require the actor's movement express some game condition or emotion. In these scenarios, the actor's movement relative to animation might prioritize visual accuracy.
+
+The remainder of this tutorial focuses on data driven root motion. You'll learn how to determine if an actor is properly set up for data driven root motion, how to extract root motion if necessary, and how to enable root motion in [Animation Editor](/docs/user-guide/visualization/animation/animation-editor/user-interface).
```

Comment:
```suggestion
The remainder of this tutorial focuses on data driven root motion. You'll learn how to determine if an actor is properly set up for data driven root motion, extract root motion if necessary, and enable root motion in [Animation Editor](/docs/user-guide/visualization/animation/animation-editor/user-interface).
```
You could drop the second and third occurrence of "how to" if you think it sounds better.